baseframe/__init__.py

```diff
@@ -248,13 +249,13 @@ def init_app(
         app.assets.register('css_all', css_all)
         app.register_blueprint(self, static_subdomain=subdomain)
 
-        for module_name in app.config.get('PRIVATE_ASSETS', []):
+        for module_name in asset_modules:
             try:
                 module = __import__(module_name)
-                module.blueprint.init_app_assets(app, assets)
+                module.blueprint.init_app(app)
                 app.register_blueprint(module.blueprint)
```

Comment:
My sense that these should be registered as a subpath of `/_baseframe`, to avoid taking up additional top-level names (as the top-level is also used by profiles). However, it's unclear if Flask routing will handle this correctly, so test it before committing this change. The static subdomain is also necessary if the app is using it.

```python
app.register_blueprint(module.blueprint, url_prefix='/_baseframe', static_subdomain=subdomain)
```

Reply:
Tried this. Doesn't seem to work as expected.